docs/getting-started/quickstart.md

```diff
@@ -57,6 +57,20 @@ Follow prompts to configure your database and create your first admin user.
 
 To start Directus locally, run `npx directus start`.
 
+<details>
+<summary>Notes on Additional Dependencies</summary>
+
+To prevent potential errors during installation, please run the following command specific to your operating system:
```

Comment:
I would not go so far as to recommend the installation of these tools in every case, but rather only after errors occured during installation.

Reply:
That makes sense; I can tweak it to something like "In case you run into errors during installation, run the following command specific to your operating system"

Reply:
That sounds perfect 👍